sonar-html-plugin/src/main/java/org/sonar/plugins/html/checks/comments/CommentUtils.java

```diff
@@ -26,6 +26,9 @@ private CommentUtils() {
   }
 
   static int lineNumber(CommentNode node, int offset) {
+    if (offset < 0) {
+      throw new IllegalArgumentException("Negative offset: " + offset);
```

Comment:
Message is meaningless and untested. Please have a message that will help another developer looking at the stacktrace understand the issue.